src/stylesheets/components/_header.scss

```
@@ -125,7 +125,7 @@ $z-index-overlay: 400;
   @include u-pin("all");
   position: fixed;
   background: color("black");
```

```suggestion
  background: color("black-transparent-70");
```

Let's use the same background color as modal overlay.

---

updated